src/web-stack/apps/api/src/app/users/entities/affirmation.entity.ts

```diff
@@ -0,0 +1,38 @@
+import { Entity, PrimaryGeneratedColumn, Column, OneToMany, ManyToOne } from 'typeorm';
+import { UserEntity } from './user.entity';
+import { LikeEntity } from './like.entity';
+import { ReaffirmationEntity } from './reaffirmation.entity';
+
+/**
+ * TODO
+ * - Time created (how to generate and store this?)
```

Comment:
Check this out:
https://github.com/typeorm/typeorm/blob/master/docs/decorator-reference.md#column-decorators